app/javascript/dashboard/featureFlags.js

```diff
@@ -18,4 +18,5 @@ export const FEATURE_FLAGS = {
   AUDIT_LOGS: 'audit_logs',
   INSERT_ARTICLE_IN_REPLY: 'insert_article_in_reply',
   INBOX_VIEW: 'inbox_view',
+  SLA: 'sla',
```

Comment:
do we need this?

Reply:
Yes, it was referenced in another PR.

https://github.com/chatwoot/chatwoot/pull/8777

https://github.com/chatwoot/chatwoot/blob/eeb7b46b02e2ad80237b03f51c888367cc0512a9/app/javascript/dashboard/components/layout/config/sidebarItems/settings.js#L168